tests/dsl/dtd/dtd_test_new_tile_cuda_kernels.cu

```diff
@@ -0,0 +1,90 @@
+#include <stdio.h>
+
+extern "C" {
+void dtd_test_new_tile_init(int *dev_data, int nb, int idx);
+void dtd_test_new_tile_sum_add(int *dev_data, int nb, int idx, int *acc);
+void dtd_test_new_tile_multiply_by_two(int *dev_data, int nb, int idx);
+}
+
+__global__ void dtnt_init(int *dev_data, int nb, int idx)
+{
+    for(int i = 0; i < nb; i++)
+        dev_data[i] = i;
+    (void)idx;
+}
+
+void dtd_test_new_tile_init(int *dev_data, int nb, int idx)
+{
+#if DEBUG_CUDA_KERNEL
+    cudaError_t err;
+    err = cudaDeviceSynchronize();
+    if( cudaSuccess != err ) {
+        printf("Error before init(%d): %d/%s\n", idx, err, cudaGetErrorString(err));
+    }
+#endif
+    dtnt_init<<<1, 1>>>(dev_data, nb, idx);
+#if DEBUG_CUDA_KERNEL
+    err = cudaDeviceSynchronize();
```

Comment:
Why are we explicitly managing the device synchronization in user code ?

Reply:
Fixed in 9b83c9b5cb0cc057c6697767308364e5c5c28e0f